client/lib/happychat/connection.js

```diff
@@ -81,6 +81,18 @@ class Connection extends EventEmitter {
 		);
 	}
 
+	sendLog( message ) {
```

Comment:
Is `Log` here and as the type below sufficiently descriptive? Could we make it more meaningful?